main.go

```diff
@@ -52,13 +53,22 @@ func RunServer(logfile io.Writer) {
 	host := pflag.String("host", "0.0.0.0", "Host of the server")
 	tlsCertPath := pflag.String("tls-cert-path", "server.crt", "TLS Certificate path")
 	tlsKeyPath := pflag.String("tls-key-path", "server.key", "TLS Private key path")
+	watchmanHost := pflag.String("watchman-host", "192.168.122.1", "Watchman Metrics Host")
+	watchmanPort := pflag.String("watchma-port", "9125", "Watchman Metrics port")
 
 	pflag.Parse()
 
 	if *authTokenSecret == "" {
 		log.Fatal("Auth token is empty")
 	}
 
+	// Initialize watchman
+	err := watchman.Configure(*watchmanHost, *watchmanPort, "agent.prod")
```

Comment:
`agent.prod` should also be a parameter.

Users of the Agent should be free to define the namespace in which this Agent is running. Assuming it is always prod is not a good practice.